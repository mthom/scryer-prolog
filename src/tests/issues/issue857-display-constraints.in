X = 1.
use_module(library(dif)).
X = 1.
dif(X,1).
halt.
