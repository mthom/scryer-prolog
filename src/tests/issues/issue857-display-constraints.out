X = 1.
true.
X = 1.
dif:dif(X,1).
